examples/layer2/aws.yml

```
@@ -0,0 +1,55 @@
+---
+# NOTE: this playbook should be run _after_:
+# 1. Running the metal.yml playbook
+# 2. Redeeming the Fabric service token in the Fabric portal
+# 3. Accepting the Direct Connect request in the AWS console
+- name: Equinix Layer 2 example -- AWS resources
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Include the required variables
+      include_vars: "vars/vars.yml"
+
+    - name: create a VPC
+      amazon.aws.ec2_vpc_net:
+        name: ansible-equinix-layer2-example
+        cidr_block: "{{ aws_network_cidr }}"
+        region: "{{ aws_region }}"
+      register: created_vpc
+
+    - name: Gather information about any VPC route table within VPC with ID "{{ created_vpc.vpc.id }}"
+      amazon.aws.ec2_vpc_route_table_info:
+        filters:
```

```suggestion
        region: "{{ aws_region }}"
```
The step `Gather information about any VPC route table within VPC with ID` failed until I added the above.

---

Added in the latest commit